app/src/data/schemas/collectionsSchema.ts

```diff
@@ -0,0 +1,48 @@
+const collectionsSchema = {
+  response: {
+    numResults: "6",
+    page: "1",
+    perPage: "40",
+    // move from "collection" to "collections"
+    collections: [
+      {
+        title: "string",
+        uuid: "string",
+        url: "string",
+        imageID: "string",
+        numberOfDigitizedItems: "string", // remove numItems
+        containsAVMaterial: "boolean",
+        constainsOnSiteMaterials: "boolean",
```

Comment:
These will always be either `true` or `false` right? So a `null` or empty value won't be the same as `false`?

Reply:
good call, "false" is sufficient